src/components/hooks/useAccordion/useAccordion.ts

```diff
@@ -2,6 +2,40 @@ import { useRef, useState, Dispatch, SetStateAction } from 'react';
 import { isArrowDown, isArrowUp } from '../../../helpers/keyboard-keys';
 import { mod } from '../../../helpers/utils';
 
+interface IAccordionHeaderProps {
+  'aria-controls': string;
```

Comment:
is there a reason why we are not extending the Button props instead of adding manually every prop?

Reply:
Same in line 25 for `div`

Reply:
this way they are required and not optional which makes your life easier when adding custom input components